metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/SequenceGeneration.java

```diff
@@ -0,0 +1,121 @@
+package com.netflix.metacat.connector.hive.sql;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.Lists;
+import com.netflix.metacat.common.server.connectors.exception.ConnectorException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowCallbackHandler;
+import org.springframework.jdbc.core.SqlParameterValue;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.sql.Types;
+import java.util.List;
+
+/**
+ * This class is used to generate the sequence ids.
+ * @author amajumdar
+ */
+@Slf4j
+@Transactional
```

Comment:
Should it be +@Transactional("hiveTxManager")

Reply:
Yes.